pages/docs/getting-started/apache.mdx

```diff
@@ -175,7 +175,7 @@ Our MySQL server should now be installed and secured. We can now move on to the
 
     Let's use APT once more to install PHP and the extensions we need for FOSSBilling.
       ```bash filename="bash"
-      sudo apt install php libapache2-mod-php php-mysql php-curl
+      sudo apt install php libapache2-mod-php php-mysql php-curl php-xml php-intl php-dom php-gd php-mbstring
```

Comment:
I'd also add `php-common`, which will add in some commonly needed files and extensions for PHP (including `curl`, so that can be removed from the install command if `php-common` is added)